OpenRA.Game/Settings.cs

```diff
@@ -129,6 +129,9 @@ public class DebugSettings
 		[Desc("Display Lua debug messages in the game chat.")]
 		public bool LuaDebug = false;
 
+		[Desc("Run map scripts in a sandbox environment for safety.")]
```

Comment:
If someone sets this to `false` once, do they need to set it to `true` again to actually be safe in the future (after testing a script of theirs)?

Can/should we warn, annoyingly?, if this is `false` somehow?

Reply:
Thats a good point. We could define this as a launch argument that is not persisted in the settings.

Reply:
Sounds alright for additional safety. Still convenient as it is at least persisted through map changes. Updated.